src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionSettings.cs

```diff
@@ -67,6 +62,12 @@ internal sealed class HttpConnectionSettings
 
         internal IDictionary<string, object?>? _properties;
 
+        // Http2 flow control settings:
+        internal bool _disableDynamicHttp2WindowSizing = DisableDynamicHttp2WindowSizing;
+        internal int _maxHttp2StreamWindowSize = MaxHttp2StreamWindowSize;
+        internal double _http2StreamWindowScaleThresholdMultiplier = Http2StreamWindowScaleThresholdMultiplier;
+        internal int _initialHttp2StreamWindowSize = Http2Connection.DefaultInitialWindowSize;
```

Comment:
It should, yeah.

I also wonder whether 64K is still the right value for this. Should we make this larger?

Reply:
I kept it 64K because in the (almost) 0 RTT loopback communication case it seemed to be sufficient according to my benchmharks:

![image](https://user-images.githubusercontent.com/6835152/123847616-14b7d800-d917-11eb-9479-20e1392d2b28.png)

From what I've seen, the window scales fast, so setting the initial value to 128K or 256K doesn't seem to bring significant benefits.

Reply:
Actually it wasn't optimal in any of my runs. Data from `0ms RTT / 5Gbps` (loopback communication) benchmarks suggests it is worth to set it to 256K, however the window will scale faster to larger than optimal maximum sizes.

I'm a bit hesitant to change the defaults without having more data. Maybe we should invest a couple of days to find the right defaults for the August preview?